src/project-1.js

```diff
@@ -1,140 +1,121 @@
 // Do not change any of the funcxtion names
 
-const multiplyByTen = (num) => {
+const multiplyByTen = num => num * 10;
   // return num after multiplying it by ten
   // code here
-};
 
-const subtractFive = (num) => {
+const subtractFive = num => num - 5;
   // return num after subtracting five
   // code here
-};
 
-const areSameLength = (str1, str2) => {
+const areSameLength = (str1, str2) => str1.length === str2.length;
   // return true if the two strings have the same length
   // otherwise return false
   // code here
-};
 
-const areEqual = (x, y) => {
+const areEqual = (x, y) => x === y;
   // return true if x and y are the same
   // otherwise return false
   // code here
-};
 
-const lessThanNinety = (num) => {
+
```

Comment:
Be careful about adding extra lines.  Spacing and layout is pretty important.  Your senior devs at a future job will be super strict about this stuff